### PR TITLE
Add component deps to main CF package.json

### PR DESCRIPTION
**Pull request by contolini**
Fixes https://github.com/cfpb/capital-framework/issues/312.

## Additions

- Component dependencies are now included in the main `package.json`.

## Removals

- No more npm postinstall script.

## Testing

Here's a little test repo that [points to this branch](https://github.com/contolini/cf-test/blob/master/package.json#L23):

```
git clone git@github.com:contolini/cf-test.git
cd cf-test
npm install
gulp build
npm start
```

Open http://127.0.0.1:8080/ and there should be a simple CF demo page with nothing broken.

## Review

- @anselmbradford 
- @marteki

### CODE REVIEW

**Comment by jimmynotjim** [comment]
👍 Working for me

File: package.json [hunk 1]
```diff
@@ -76,7 +75,10 @@
     "wcag": "^0.2.2"
   },
   "dependencies": {
-    "async": "^1.5.0",
-    "npmi": "^1.0.1"
+    "box-sizing-polyfill": "0.1.0",
```
Comment (anselmbradford) [inline package.json]: We should deprecate the need for this. It's for IE7-

**Comment by anselmbradford** [comment]
Icons are 404 on the demo page, but works for me too 👍

**Comment by contolini** [comment]
@anselmbradford Good catch. I had a typo in `cf-test`'s gulpfile and the icons weren't being copied over to `dist/`.

**Pull request merged**